pkgs/top-level/all-packages.nix

```diff
@@ -35692,6 +35692,8 @@ with pkgs;
 
   yabasic = callPackage ../development/interpreters/yabasic { };
 
+  twiggy = callPackage ../development/tools/twiggy { };
```

Comment:
I guess this has to be added to the correct alphabetically sorted position?

Reply:
Done